src/pages/admin/DynamicForm/FormRegister.js

```diff
@@ -376,6 +376,13 @@ const FormRegister = (props) => {
     }
   }, [currEvent]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const emailValidation = (value) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
can you use EMAIL_REGEX from registration.js instead

Reply:
Great suggestion, it's worth noting that the root cause of this issue is that the backend's regex validation is not the same as the regex validation we have on the frontend. And because the error is being thrown from the backend, we should make the frontend match what we have on the backend

Thankfully the EMAIL_REGEX we have in registration.js happens to be the same as the backend's regex, so we should eventually start using EMAIL_REGEX everywhere on `bt-web` and `bt-web-v2`

(why we never used EMAIL_REGEX is beyond me)

Backend code for reference: https://github.com/ubc-biztech/serverless-biztechapp/blob/2a84faad0b729928bc92473c7a3be4bdb52e2302/lib/utils.js#L36-L39

Reply:
Hey there, tried to implement the solution and thanks everyone for the suggestion. Came up with this solution:
![image](https://github.com/user-attachments/assets/1db7c06c-4885-4107-9268-f07258ad104e)